module/input/NatNet/src/NatNet.cpp

```diff
@@ -402,10 +403,15 @@ namespace module::input {
 
 
     void NatNet::process(const std::vector<char>& input) {
-
         // Get our packet
         const Packet& packet = *reinterpret_cast<const Packet*>(input.data());
 
+        // std::stringstream ss;
+        // for (const auto& c : input) {
+        //     ss << std::setw(2) << std::setfill('0') << std::hex << int(c);
+        // }
+        // NUClear::log(ss.str());
+
```

Comment:
remove comented out code

Reply:
Replace commented code with a config option and ability to write the raw packets to files for future debugging